server/routes/health.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 
-const createHealthRouter = ({ buildInfo:build }) => {
+const createHealthRouter = ({ buildInfo: build }) => {
```

Comment:
having to format this as a commit managed to get in that escaped the linting check